src/packages/ganache/package.json

```diff
@@ -57,11 +56,10 @@
     "truffle"
   ],
   "devDependencies": {
-    "@ethereumjs/vm": "6.2.0",
     "@ganache/cli": "0.6.0",
     "@ganache/core": "0.6.0",
     "@ganache/flavors": "0.6.0",
-    "@microsoft/api-extractor": "7.31.1",
+    "@microsoft/api-extractor": "7.20.1",
```

Comment:
This is reverting back to the version that's in develop. Upgrading broke bundling of levelup